app/data/cy/census_household.json

```diff
@@ -0,0 +1,4786 @@
+    "eq_id": "census",
+    "groups": [
+        {
+            "id": "who-lives-here",
+            "title": "Pwy sy’n byw yma?",
+            "completed_id": "who-lives-here-completed",
+            "highlight_when": [
+                "who-lives-here-relationship",
+                "who-lives-here-interstitial"
+            ],
+            "blocks": [
+                {
+                    "type": "questionnaire",
+                    "id": "permanent-or-family-home",
+                    "title": "Who lives here?",
+                    "sections": [
+                        {
+                            "id": "permanent-or-family-home-section",
+                            "title": "Pwy sy’n byw yma?",
+                            "questions": [
+                                {
+                                    "id": "permanent-or-family-home-question",
+                                    "title": "A oes unrhyw un yn byw yma fel eu cartref parhaol neu gartref y teulu?",
+                                    "number": "1",
+                                    "guidance": [
+                                        {
+                                            "title": "Dylech gynnwys",
+                                            "list": [
+                                                "Chi’ch hun, os mai hwn yw eich cartref parhaol neu gartref y teulu",
+                                                "Aelodau o’r teulu, gan gynnwys partneriaid, plant a babanod a anwyd ar 9 Ebrill 2017 neu cyn y dyddiad hwnnw",
+                                                "Myfyrwyr a/neu blant ysgol sy’n byw oddi cartref yn ystod y tymor",
+                                                "Tenantiaid, lojers neu bobl sy’n rhannu cartref"
+                                            ]
+                                        }
+                                    ],
+                                    "type": "General",
+                                    "answers": [
+                                        {
+                                            "id": "permanent-or-family-home-answer",
+                                            "mandatory": true,
+                                            "guidance": "<p>I’r rhan fwyaf o bobl, eu cartref parhaol fydd y cyfeiriad lle maent yn treulio’r rhan fwyaf o’r amser.</p>",
+                                            "options": [
+                                                {
+                                                    "label": "Oes",
+                                                    "value": "Yes"
+                                                },
+                                                {
+                                                    "label": "Nac oes",
+                                                    "value": "No",
+                                                    "description": "Er enghraifft ail gyfeiriad neu dŷ gwyliau yw hwn"
```

Comment:
Same with other accented characters elsewhere in the schema. 
![image](https://cloud.githubusercontent.com/assets/4097796/21317348/a0df9fc6-c5fc-11e6-9e52-51a53143900a.png)